libs/agno/agno/tools/mcp.py

```diff
@@ -388,6 +391,62 @@ async def initialize(self) -> None:
             log_debug(f"{self.name} initialized with {len(filtered_tools)} tools")
             self._initialized = True
 
+            # ===== PROMPTS SECTION (new code) =====
+            try:
+                # Get the list of prompts from the MCP server
+                available_prompts = await self.session.list_prompts()
+                
+                # Store prompts in a dictionary for easy access
+                self.prompts = {}
+                
+                for prompt in available_prompts.prompts:
+                    try:
+                        # Store the prompt metadata
+                        self.prompts[prompt.name] = prompt
+                        
+                        # Get an entrypoint for the prompt (same pattern as tools!)
+                        entrypoint = get_entrypoint_for_prompt(prompt, self.session)
+                        
+                        # Build parameters schema from prompt arguments
+                        parameters = {
+                            "type": "object",
+                            "properties": {},
+                            "required": []
+                        }
+                        
+                        # Add each argument from the prompt to the schema
+                        if prompt.arguments:
+                            for arg in prompt.arguments:
+                                parameters["properties"][arg.name] = {
+                                    "type": "string",
+                                    "description": arg.description if arg.description else f"Argument: {arg.name}"
+                                }
+                                if arg.required:
+                                    parameters["required"].append(arg.name)
+
+                        # Create a Function for the prompt (just like tools!)
+                        f = Function(
+                            name=f"{prompt.name}",
```

Comment:
This should be a something like `get_{prompt.name}` because we are basically telling the agent "here is a tool to get a prompt". 
Right?